### ISSUE TITLE
Is there any error in discriminator loss in GAIL?

### ISSUE THREAD

**Issue by Joywanglulu** [p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch#74]
I'm wondering if the loss of discriminator in GAIL could be 
"discrim_loss = discrim_criterion(g_o, zeros((states.shape[0], 1), device=device)) + \
            discrim_criterion(e_o, ones((expert_traj.shape[0], 1), device=device))"

instead of 
"discrim_loss = discrim_criterion(g_o, ones((states.shape[0], 1), device=device)) + \
            discrim_criterion(e_o, zeros((expert_traj.shape[0], 1), device=device))"?
